src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusTopicAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <inherit/>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class ServiceBusTopicAttribute : ServiceBusAttribute
```

Comment:
>ServiceBusTopicAttribute [](start = 24, length = 24)

What's the guidance here. Is the base class now for queues and the new derived class for topics?  (but we still need to allow the existing SBA to work on Topics for backwards compat?)